### PR TITLE
Feature/annie html exercise3

### PR DESCRIPTION
**Pull request by Annie2106**
Hello, I want to share with you the exercise 3. Thanks

### CODE REVIEW

File: 4-forms-validations/AnnieFormEx3.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<p>Welcome to PintaYa Form
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:1]: you're missing the basic structure of the html document. Don't forget to include `<html>, <head>, <title>, <body>`

File: 4-forms-validations/AnnieFormEx3.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    <br>
+    <img heiht = " 100" width = "100" src ="C:\Users\Bivi Anne\Documents\push-dev\html-tutorial\4-forms-validations/pinta.jpg  " alt = "PintaYa"/>
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:3]: You don't need spaces in between attributes and values. For the src, you can do:
```suggestion
    <img height="100" width="100" src="pinta.jpg" alt="PintaYa"/>
```

File: 4-forms-validations/AnnieFormEx3.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+</p>
+<p>Dear costumer, please fill in the requested data, in order you to have your products as soon as possible</p>
+<form action="">
+    <label for="personal information">Personal Information</label>
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:7]: These should not be a label but a heading or a paragraph.

File: 4-forms-validations/AnnieFormEx3.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    <fieldset>
+        <legend>Please, check and fill next blanck spaces:</legend>
+    <label for="first name">First Name</label>
+    <input type= "text" id= "name" name= "user_name" >
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:11]:
```suggestion
    <label for="name">First Name</label>
    <input type= "text" id= "name" name= "user_name" >
```
The attribute for should match the id of the input it is related to.

File: 4-forms-validations/AnnieFormEx3.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        <label for= "other" > Advanced web development </label>
+
+    </fieldset>
+    <br>
+
+    <label for="Contact Information">Contact Information</label>
+   <fieldset>
+        <legend>Conatact Information</legend>
+    <label for="celular number">Celular Conatct Number</label>
+    <input type="number" id= "celular number" name= "user_number">
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:44]: you're missing the required attribute.

```suggestion
    <input type="number" id= "celular number" name= "user_number" required>
```

File: 4-forms-validations/AnnieFormEx3.html [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    <abbr title= "required" aria-label= "required" > * </abbr>
+    <br>
+
+    <label for="e-mail">e-mail</label>
+    <input type="email" name="e-mail" required="required" oninvalid="setCustomValidity('share us your email')" oninput="setCustomValidity('')">
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:49]: this is great, you put your custom message!

File: 4-forms-validations/AnnieFormEx3.html [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+    <label for="pkind of person">Kind of Person</label>
+    <abbr title= "required" aria-label= "required" > * </abbr>
+    <fieldset>
+        <legend>Please, select what kin of person you are</legend>
+        <label for="Personal Status">Personal Status</label>
+        <select name="avaible procducts" id="products"> 
+            <datalist id="list1">
+            <option value="natural">Natural</option>
+            <option value="legal">Legal</option>
+        </datalist>
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:63]: nice to see you used a `<datalist>`

File: 4-forms-validations/AnnieFormEx3.html [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+        </select>
+    </fieldset>
+    <br>
+
+
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:68]: these spaces can be removed.

File: 4-forms-validations/AnnieFormEx3.html [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+    </fieldset>
+    <br>
+
+    <p>
+        <label for= "t" > Please, if you have any comment or sugestion, leave your messge </label>
+        <br>
+        <textarea id= "t" name= "msg" maxlength= "500" rows= "10" ></textarea>
+    </p>
+    <br>
+    <input type="submit" value="Submit">
```
Comment (pushdev-code) [inline 4-forms-validations/AnnieFormEx3.html:111]:
```suggestion
    <input value="Submit">   
```

**Review by pushdev-code — CHANGES_REQUESTED** [review]
Please double check the `for` attribute values in your labels, They should match their corresponding input id. 
Use input ids without spaces.
Please include the basic structure for an HTML document.

**Comment by Annie2106** [comment]
Sorry! I made a huge mistake, that's why I have a new one.

**Pull request closed without merging**